sdk/monitor/Azure.Monitor.Query/src/LogsClient.cs

```diff
@@ -36,24 +47,54 @@ public LogsClient(TokenCredential credential, LogsClientOptions options)
             _rowBinder = new RowBinder();
         }
 
+        /// <summary>
+        /// Initializes a new instance of <see cref="LogsClient"/> for mocking.
+        /// </summary>
         protected LogsClient()
         {
         }
 
+        /// <summary>
+        /// Executes the logs query.
+        /// </summary>
+        /// <param name="workspaceId">The workspace to include in the query.</param>
+        /// <param name="query">The query text to execute.</param>
+        /// <param name="timeSpan">The timespan over which to query data.</param>
```

Comment:
Does this filter to log entries created from (now - timespan) to now?

Reply:
Yes

Reply:
Should clarify in the comment.